middleware/encryptcookie/README.md

```diff
@@ -70,7 +70,7 @@ type Config struct {
 	// Base64 encoded unique key to encode & decode cookies.
 	//
 	// Required. Key length should be 32 characters.
```

Comment:
Can you update this comment to mention that the key length should be 32 bytes _after_ base64-decoding:

`Required. The key should be 32 bytes of random data in base64-encoded form.`

Reply:
Should be done in other parts of the README and the code itself as well.

Reply:
```suggestion
	// Required. The key should be 32 bytes of random data in base64-encoded form.
```